tags_add/tests/specs/missing_first_pilot_tag.yml

```diff
@@ -0,0 +1,21 @@
+initialization:
+  steps:
+    - type: commit
+      empty: true
+      message: Add January duty roster
+      id: start
+    - type: commit
+      empty: true
+      message: Update duty roster for February
+    - type: commit
+      empty: true
+      message: Update roster for March
+    - type: tag
+      tag-name: v1.0
```

Comment:
We don't currently support creating lightweight tags from `repo-smith`. Let me add that support first before we proceed with this